app/controllers/help_with_fees_controller.rb

```diff
@@ -0,0 +1,20 @@
+class HelpWithFeesController < ApplicationController
+  def show
+    @fee = Fee.find(params[:id])
+  end
+
+  def update
+    @fee = Fee.find(params[:id])
+    if @fee
+      @fee.update_column(:help_with_fees_reference, help_with_fees_params)
```

Comment:
As per our conversation-it's because I'm not using the whole object.  The form tag is only sending `help_with_fees_reference`.